x11-wm/awesome/awesome-4.3-r1.ebuild

```diff
@@ -54,13 +53,15 @@ DEPEND="${RDEPEND}
 		dev-lua/luacheck
 	)
```

Comment:
can you check the build system here? My gut feeling is that

- app-text/asciidoc
- app-text/xmlto
- dev-util/gperf
- media-gfx/imagemagick[png]
- doc? ( dev-lua/ldoc )
- and app-shells/zsh in test

can likely also go into `BDEPEND`

Reply:
It seems off to me as well, will check it

Reply:
Reworked it a little bit, have a look. Version specifiers are outdated and always satisfied so I dropped them. Looks like `gperf` and `xmlto` are not used anymore, there might be more incorrect deps